src/include/migraphx/op/multibroadcast.hpp

```diff
@@ -37,8 +37,10 @@ namespace op {
 /**
  * Broadcast multiple dimensions between two tensors.
  * Two versions of this operator: one input and two inputs.
- * One input version uses output_lens attribute and broadcasts to it.
- * Two inputs version broadcasts both inputs to the common shape at evaluation time.
+ * One input version uses output_lens attribute and broadcasts to it (does not support
+ * dynamic shape input)).
+ * Two inputs version broadcasts to the shape of the second input at evaluation time.
```

Comment:
Not exactly true, the previous statement was correct. It is possible that the common shape is the shape of neither of the inputs.